slices/libx265-199.yaml

```diff
@@ -0,0 +1,11 @@
+package: libx265-199
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
+      - libnuma1_libs
+      - libstdc++6_libs
+    contents:
+      /usr/lib/*-linux-*/libx265.so.199*:
```

Comment:
I don't think we need the wildcard at the end here, being the only file there is.

Reply:
y